File: Leetcode/2011. Final Value of Variable After Performing Operations/solution.c
int finalValueAfterOperations(char ** operations, int operationsSize){
    int acum = 0;
    for(int i = 0; i < operationsSize; i++) {
        acum += (operations[i][0] == '+' || operations[i][1] == '+') ? 1 : -1;
    }
    
    return acum;
}

File: Leetcode/2011. Final Value of Variable After Performing Operations/solution.go
import ("strings");


func finalValueAfterOperations(operations []string) int {
    acum := 0
    
    for _, op := range(operations) {
        if strings.Contains(op, "+") {
            acum += 1
        } else {
            acum -= 1
        }
    }
    
    return acum
}